public/app/features/dashboard/components/PanelEditor/VisualizationTab.tsx

```diff
@@ -30,9 +30,6 @@ export const VisualizationTabUnconnected = React.forwardRef<HTMLInputElement, Pr
     const theme = useTheme();
     const styles = getStyles(theme);
 
-    if (!plugin) {
```

Comment:
The condition here meant that the below hooks were implicitly being called conditionally, breaking one of the [rules of hooks](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level), which was causing an error.